src/Workspaces/Core/Portable/Remote/RemoteSupportedLanguages.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Remote
+{
+    internal static class RemoteSupportedLanguages
```

Comment:
consider: "Remote" in type name might be redundant in this namespace (not sure if it conflicts with another "SupportedLanguages" without though).